src/index.tsx

```diff
@@ -3,19 +3,24 @@ import React from 'react';
 import ReactDOM from 'react-dom/client';
 import { Provider } from 'react-redux';
 import { BrowserRouter } from 'react-router-dom';
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
 import { store } from './store/store';
 
+const queryClient = new QueryClient();
+
 const root = ReactDOM.createRoot(document.getElementById('root') as HTMLElement);
 root.render(
   <React.StrictMode>
-    <GlobalStyle />
-    <Provider store={store}>
-      <BrowserRouter>
-        <App />
-      </BrowserRouter>
-    </Provider>
+    <QueryClientProvider client={queryClient}>
+      <GlobalStyle />
+      <Provider store={store}>
```

Comment:
provider가 모여있는게 더 보기 좋을 것 같아요
```suggestion
    <GlobalStyle />
    <QueryClientProvider client={queryClient}>
      <Provider store={store}>
```

Reply:
넵 수정했습니다!